src/controllers/tasks/active.officers.details.controller.ts

```diff
@@ -43,6 +45,24 @@ const buildSecretaryList = (officers: ActiveOfficerDetails[]): any[] => {
     });
 };
 
+const buildCorporateSecretaryList = (officers: ActiveOfficerDetails[]): any[] => {
+  return officers
```

Comment:
Could rename this to buildCorporateOfficerList as it can be reused for corporate directors. We can see what people think about this.

Reply:
True it seems the fields are identical so this seems like a good shout 👍